src/index.ts

```diff
@@ -0,0 +1,83 @@
+import { bytes_to_hex, Sha256 } from "asmcrypto.js"
+import {
+  TFileMetadata,
+  TdrFile,
+  TGenerateMetadata,
+  TTotalChunks,
+  TSliceToArray,
+  TChunkProgressFunction
+} from "./types"
+
+export const extractFileMetadata: TFileMetadata = async (
+  files,
+  progressFunction,
+  chunkSize = 100000000
```

Comment:
Could this be `chunkSizeBytes`. It will probably save some confusion.

Reply:
Yeah, makes sense